backend/src/packages/auth/auth.controller.ts

```diff
@@ -75,6 +94,28 @@ class AuthController extends BaseController {
       payload: await this.authService.signUp(options.body),
     };
   }
+
+  private async signIn(
+    options: APIHandlerOptions<{
+      body: UserSignInRequestDto;
+    }>,
+  ): Promise<APIHandlerResponse> {
+    const user = await this.authService.verifyLoginCredentials(options.body);
+
+    if (!user) {
+      return {
+        status: HTTPCode.UNAUTHORIZED,
+        payload: { message: 'Unauthorized' },
+      };
+    }
+
+    await this.authService.signIn(user.id);
```

Comment:
May be I'm missing smth, but as I see you already got user above by email from userRepo. Therefore it's no reason in finding user again, because you already have it and it definitely will be in db

Reply:
I did it by example from the [mini-project](https://github.com/BinaryStudioAcademy/thread-js/blob/production/server/src/packages/auth/auth.service.js), if its not needed - I will change it.

Reply:
Oh, I see. In mini-project by making this step they're creating token and returning it with the user, so maybe it's supposed to be the same way here. Although pr of an issue #16 is already merged, I'm not sure who should implement jwt token in sign in/sign up, because this action is unchecked there

Reply:
Great point, bcos as I see no token and jwt was used

Reply:
Should I delete signIn and findById methods too then, if they are not used atm?